Gems/Atom/Tools/MaterialEditor/Code/Source/main.cpp

```diff
@@ -28,17 +28,7 @@ int main(int argc, char** argv)
     QApplication::setOrganizationDomain("o3de.org");
     QApplication::setApplicationName("O3DE Material Editor");
```

Comment:
Can these lines be deleted now that they are in the base application?
    QApplication::setOrganizationName("O3DE");
    QApplication::setOrganizationDomain("o3de.org");
    QApplication::setApplicationName("O3DE Material Editor");